diff --git a/node_modules/next/dist/build/webpack-config.js b/node_modules/next/dist/build/webpack-config.js
index c954ebe..964df2c 100644
--- a/node_modules/next/dist/build/webpack-config.js
+++ b/node_modules/next/dist/build/webpack-config.js
@@ -73,7 +73,7 @@ if(res.match(/node_modules[/\\]webpack/)||res.match(/node_modules[/\\]css-loader
 if(isNextExternal||res.match(/node_modules[/\\].*\.js$/)){const externalRequest=isNextExternal?// Generate Next.js external import
 _path.default.posix.join('next','dist',_path.default.relative(// Root of Next.js package:
 _path.default.join(__dirname,'..'),res)// Windows path normalization
-.replace(/\\/g,'/')):request;return callback(undefined,`commonjs ${externalRequest}`);}// Default behavior: bundle the code!
+.replace(/\\/g,'/')):request; if(/^react$/.test(externalRequest)) return callback(undefined, 'commonjs ' + externalRequest); return callback();}// Default behavior: bundle the code!
 callback();}let webpackConfig={externals:!isServer?// make sure importing "next" is handled gracefully for client
 // bundles in case a user imported types and it wasn't removed
 // TODO: should we warn/error for this instead?
